db/queries/licenses.sql

```diff
@@ -1,23 +1,44 @@
 -- name: InsertLicense :one
-INSERT INTO licenses (guid, file, key)
-VALUES (?, ?, ?)
+INSERT INTO licenses (pool_id, guid, file, key)
+VALUES (?, ?, ?, ?)
 RETURNING *;
 
 -- name: GetLicenseByGUID :one
 SELECT *
 FROM licenses
 WHERE guid = ?;
 
+-- name: GetUnpooledLicenseByGUID :one
+SELECT *
+FROM licenses
+WHERE guid = ? and pool_id IS NULL;
+
+-- name: GetPooledLicenseByGUID :one
```

Comment:
Need to iron out the naming conventions here because we're mixing `GetX` and `GetUnpooledX` in quite a few places, and I'd like to be more explicit as to what queries use `pool_id IS NULL` vs `pool_id = ?` vs no pool predicate at all.

Reply:
Maybe something like:

- `GetLicenseByGUID`: retrieve license by GUID regardless of pool.
- `GetLicenseWithoutPoolByGUID`: license with a null pool.
- `GetLicenseWithPoolByGUID`: license for a specific pool.